html/webappapis/system-state-and-capabilities/the-navigator-object/protocol-handler-fragment-nosw.https.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<!-- Use a non-UTF-8 encoding to see how the handler URL is parsed -->
+<meta charset=windows-1254>
+<meta name=timeout content=long>
+<title>registerProtocolHandler() and a handler with %s in the fragment (does not use a service worker)</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src=/service-workers/service-worker/resources/test-helpers.sub.js></script>
+<script>
+'use strict';
+promise_setup(async () => {
+  await test_driver.set_rph_registration_mode("autoaccept");
+  await test_driver.bless('handler registration');
```

Comment:
Does this argument do anything? Sorry, not familiar with `test_driver`, but this looks awfully generic.

Reply:
Umm, good question; I really don't know. I hope @jgraham could provide some insight. 
I've just follow the example in the WPT documentation site:

https://web-platform-tests.org/writing-tests/testdriver.html#test_driver.bless

```
test_driver.bless('initiate media playback', function () {
  mediaElement.play();
});

Reply:
Although, that suggests three arguments? Is the documentation wrong?

Reply:
> Although, that suggests three arguments? Is the documentation wrong?

Well, the third argument is for the Browsing context, which if null indicates the use of the "current" browsing context.